src/Umbraco.Core/Runtime/SqlMainDomLock.cs

```diff
@@ -147,20 +152,18 @@ private void ListeningLoop()
                     continue;
                 }
 
-                if (!_dbFactory.Configured)
-                {
-                    // if we aren't configured, we just keep looping since we can't query the db
-                    continue;
-                }
-
```

Comment:
Based on the description of the PR, I don't understand why this is removed. Is it intentionally?

Reply:
This was duplicate code, if you expand this you can see this exists above, must have been a merge that inserted the duplicate code.